scripts/install/setup.sh

```diff
@@ -16,6 +16,21 @@ PARENT_DIR=$PARENT_DIR $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_git_co
 
 source "$PROPERTIES_FILE"
 
+LOCATION_GKE="--zone $ZONE"
+LOCATION_REDIS="--zone $ZONE"
+while getopts ":r" opt; do
+  case ${opt} in
+    r )  echo "Deployment done to a regional cluster in $REGION"
+         LOCATION_GKE="--region $REGION"
+         LOCATION_REDIS=""
+      ;;
+    \? ) echo "Usage: cmd [-r]"
```

Comment:
Should not display command-line help; the accompanying documentation in the provision-spinnaker.md file should be updated instead.